devel/smake/Portfile

```diff
@@ -1,41 +1,56 @@
-PortSystem 1.0
-PortGroup  compiler_blacklist_versions 1.0
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+name			        smake
+version			        1.6
+categories		        devel
+platforms		        darwin
+maintainers             {hotmail.com:amtor @RobK88} \
+                        openmaintainer
+license			        CDDL
+description		        highly portable make program with automake features
+long_description	    Smake is a highly portable make program with automake \
+				        features. It is currently mainly targeted to be used \
+				        with the makefiles system.
+
+homepage		        https://sourceforge.net/projects/s-make/
+master_sites            sourceforge:schilytools
+
+checksums               sha1    876b30c2f63e1ba7d682d0cf58789d26a527f070 \
+                        rmd160  9ad475b67122f487844162ea2e6e3659ea25579f \
+                        sha256  7c93f43d22c6e3de0eafe018ceb26ddd5cc6c543dc301f307d1e6ca313248aa4 \
+                        size    4952788
+
+distname                schily-2021-09-18
```

Comment:
I think that `extract.rename  yes` here should make things cleaner.

Reply:
I do not see `extract.rename` in the MacPorts Guide.  Where is so info on it and how would i use it here?

Reply:
It is a feature at MacPorts 2.8.1 :) Maybe nobody updated the guide, but it widley used at ports.